test/support/api_case.ex

```diff
@@ -110,16 +110,17 @@ defmodule CodeCorps.ApiCase do
         conn |> post(path, payload)
       end
 
-      def request_update(conn, :not_found), do: conn |> request_update(-1, %{})
-      def request_update(conn, attrs \\ %{}), do: conn |> request_update(default_record, attrs)
+      def request_update(conn), do: request_update(conn, %{})
```

Comment:
Rather than trying to cover the varying arity, what are your thoughts on keeping the default on line 115? Having `request_update/1` call `request_update/2` seems like the same idea as a default value.

`def request_update(conn, attrs // %{}), do: request_update(conn, default_record, attrs)`